packages/now-routing-utils/src/schemas.ts

```diff
@@ -1,3 +1,27 @@
+const hasSchema = {
+  type: 'array',
+  maxItems: 1024,
```

Comment:
I think this should probably be `12` or something around that

Reply:
Yeah or maybe 16 if you want to keep it base 2. 

1024 seems too large if you only have 3 keys to choose from.

Reply:
Updated to `16`